DPGAnalysis/HcalTools/plugins/CMTRawAnalyzer.cc

```diff
@@ -7394,532 +7880,543 @@ void CMTRawAnalyzer::beginJob() {
-    h_mapDepth4Error_HE = new TH2F("h_mapDepth4Error_HE", " ", neta, -41., 41., nphi, 0., bphi);
-    h_mapDepth5Error_HE = new TH2F("h_mapDepth5Error_HE", " ", neta, -41., 41., nphi, 0., bphi);
-    h_mapDepth6Error_HE = new TH2F("h_mapDepth6Error_HE", " ", neta, -41., 41., nphi, 0., bphi);
-    h_mapDepth7Error_HE = new TH2F("h_mapDepth7Error_HE", " ", neta, -41., 41., nphi, 0., bphi);
-    h_fiber0_HE = new TH1F("h_fiber0_HE", " ", 10, 0., 10.);
-    h_fiber1_HE = new TH1F("h_fiber1_HE", " ", 10, 0., 10.);
-    h_fiber2_HE = new TH1F("h_fiber2_HE", " ", 40, 0., 40.);
-    h_repetedcapid_HE = new TH1F("h_repetedcapid_HE", " ", 5, 0., 5.);
-
-    h_errorGeneral_HF = new TH1F("h_errorGeneral_HF", " ", 5, 0., 5.);
-    h_error1_HF = new TH1F("h_error1_HF", " ", 5, 0., 5.);
-    h_error2_HF = new TH1F("h_error2_HF", " ", 5, 0., 5.);
-    h_error3_HF = new TH1F("h_error3_HF", " ", 5, 0., 5.);
-    h_error4_HF = new TH1F("h_error4_HF", " ", 5, 0., 5.);
-    h_error5_HF = new TH1F("h_error5_HF", " ", 5, 0., 5.);
-    h_error6_HF = new TH1F("h_error6_HF", " ", 5, 0., 5.);
-    h_error7_HF = new TH1F("h_error7_HF", " ", 5, 0., 5.);
-    h_amplError_HF = new TH1F("h_amplError_HF", " ", 100, -2., 98.);
-    h_amplFine_HF = new TH1F("h_amplFine_HF", " ", 100, -2., 98.);
-    h_mapDepth1Error_HF = new TH2F("h_mapDepth1Error_HF", " ", neta, -41., 41., nphi, 0., bphi);
-    h_mapDepth2Error_HF = new TH2F("h_mapDepth2Error_HF", " ", neta, -41., 41., nphi, 0., bphi);
-    h_mapDepth3Error_HF = new TH2F("h_mapDepth3Error_HF", " ", neta, -41., 41., nphi, 0., bphi);
-    h_mapDepth4Error_HF = new TH2F("h_mapDepth4Error_HF", " ", neta, -41., 41., nphi, 0., bphi);
-    h_fiber0_HF = new TH1F("h_fiber0_HF", " ", 10, 0., 10.);
-    h_fiber1_HF = new TH1F("h_fiber1_HF", " ", 10, 0., 10.);
-    h_fiber2_HF = new TH1F("h_fiber2_HF", " ", 40, 0., 40.);
-    h_repetedcapid_HF = new TH1F("h_repetedcapid_HF", " ", 5, 0., 5.);
-
-    h_errorGeneral_HO = new TH1F("h_errorGeneral_HO", " ", 5, 0., 5.);
-    h_error1_HO = new TH1F("h_error1_HO", " ", 5, 0., 5.);
-    h_error2_HO = new TH1F("h_error2_HO", " ", 5, 0., 5.);
-    h_error3_HO = new TH1F("h_error3_HO", " ", 5, 0., 5.);
-    h_error4_HO = new TH1F("h_error4_HO", " ", 5, 0., 5.);
-    h_error5_HO = new TH1F("h_error5_HO", " ", 5, 0., 5.);
-    h_error6_HO = new TH1F("h_error6_HO", " ", 5, 0., 5.);
-    h_error7_HO = new TH1F("h_error7_HO", " ", 5, 0., 5.);
-    h_amplError_HO = new TH1F("h_amplError_HO", " ", 100, -2., 98.);
-    h_amplFine_HO = new TH1F("h_amplFine_HO", " ", 100, -2., 98.);
-    h_mapDepth4Error_HO = new TH2F("h_mapDepth4Error_HO", " ", neta, -41., 41., nphi, 0., bphi);
-    h_fiber0_HO = new TH1F("h_fiber0_HO", " ", 10, 0., 10.);
-    h_fiber1_HO = new TH1F("h_fiber1_HO", " ", 10, 0., 10.);
-    h_fiber2_HO = new TH1F("h_fiber2_HO", " ", 40, 0., 40.);
-    h_repetedcapid_HO = new TH1F("h_repetedcapid_HO", " ", 5, 0., 5.);
+    //  ha2 = fs_->make<TH2F>("ha2"," ", neta, -41., 41., nphi, 0., bphi);
```

Comment:
Is this commented out code needed?

Reply:
All the commented out code are left to remind the legacy code. Should they be removed? Please suggest

Reply:
If the legacy code is not supposed to be resurrected any more, better to remove and clean up
Alternatively, add at least a couple of further lines of comment that explains what those lines are intended to.